frontend/src/components/GDXForm/Fields/GDXCheckbox/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from "react";
+import { Checkbox } from "@mui/material";
+import { ICheckboxProps } from "../../../../types";
+
+/**
+ *
+ * @param   {{fieldValue:unknown, setFieldValue:Function, pickerData:unknown}}
+ * @returns                                                                    a JSX checkbox
+ */
+
+export const GDXCheckbox: FC<ICheckboxProps> = ({ checked, setFieldValue }) => {
+  return (
+    <>
+      <Checkbox
+        onChange={(event) => {
+          setFieldValue("is_active", event.target.checked);
```

Comment:
Line 16 is using a static value "is_active" If we want to reuse this checkbox we will need to change this to be a dynamic value

Reply:
Agreed, thanks.